extensions/wikia/UserLogin/UserSignupSpecialController.class.php

```diff
@@ -175,10 +175,17 @@ public function signupForm () {
 
 	public function captcha() {
 		$this->rawHtml = '';
-		$captchaObj = $this->getCaptchaObj();
-		if ( !empty( $captchaObj ) ) {
-			$this->rawHtml = $captchaObj->getForm();
-			$this->isFancyCaptcha = ( class_exists( 'Captcha\Module\FancyCaptcha' ) && $captchaObj instanceof Captcha\Module\FancyCaptcha );
+		$this->isFancyCaptcha = false;
+
+		if ( $this->shouldDisplayCaptcha() ) {
+			$captchaObj = $this->getCaptchaObj();
+			if ( !empty( $captchaObj ) ) {
+				$this->rawHtml = $captchaObj->getForm();
+				$this->isFancyCaptcha = (
+					class_exists( 'Captcha\Module\FancyCaptcha' ) &&
```

Comment:
this was another whitespace fix; I will address them in another ticket along with refactor